src/GitVersionCore/Core/Abstractions/Git/IObjectId.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace GitVersion
+{
+    public interface IObjectId : IEquatable<IObjectId>
```

Comment:
```suggestion
    public interface IObjectId : IEquatable<IObjectId>, IComparable<IObjectId>
```

Reply:
Agree, I will change that in a subsequent PR